odl/solvers/functional/default_functionals.py

```diff
@@ -2292,6 +2293,82 @@ def gradient(self):
                 (1 / self.sigma) * self.functional.proximal(self.sigma))
 
 
+class HuberL1L2(Functional):
+    """The functional corresponding to the Huberized L1-norm."""
+
+    def __init__(self, space, gamma):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        space : `DiscreteLp` or `FnBase`
+            Domain of the functional.
+        gamma : float
+            Smoothing parameter of Huberization. If gamma = 0, then functional
+            is non-smooth corresponds to the usual L1 norm. For gamma > 0, it
+            has a 1/gamma-Lipschitz gradient so that its convex conjugate is
+            gamma-strongly convex.
+
+        Examples
+        --------
+
+        Compare HuberL1 and L1 for vanishing smoothing gamma=0
+
+        >>> import odl
+        >>> X = odl.uniform_discr([0, 0], [1, 1], [5, 5])
+        >>> x = odl.phantom.white_noise(X)
+        >>> alpha = 2
+        >>> gamma = 0
+        >>> H = alpha * odl.solvers.HuberL1L2(X, gamma)
+        >>> L1 = alpha * odl.solvers.GroupL1Norm(X, 2)
+        >>> abs(H(x) - L1(x)) < 1e-10
+        """
```

Comment:
It would be tremendous with a `Notes` section here where you give the definition of the functional

Reply:
Done, please have a look.